pkg/util/kubernetes/kubelet/diagnosis.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package kubelet
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/diagnose/diagnosis"
+	"github.com/DataDog/datadog-agent/pkg/util/docker"
```

Comment:
As you're using docker libs, you have to add the `docker` build tag to that file too, that's what's breaking appveyor.

We'll work later in decoupling the docker and kubelet packages (to allow dockerless kubernetes integration), but that's the current state of things.